tests/ci/docker_images/build_cryptofuzz_modules.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -exo pipefail
+
+function env {
+  export "$1"="$2"
+  echo "export ${1}=\"${2}\"" >> "${FUZZ_ROOT}/fuzz_env.sh"
+}
+# Recommended flags from https://github.com/guidovranken/cryptofuzz/blob/master/docs/building.md
+export CFLAGS="-fsanitize=address,undefined,fuzzer-no-link -O2 -g"
+export CXXFLAGS="-fsanitize=address,undefined,fuzzer-no-link -D_GLIBCXX_DEBUG -O2 -g"
+
+# Setup base of Cryptofuzz
+cd "$FUZZ_ROOT"
+MODULES_ROOT="${FUZZ_ROOT}/modules"
+#git clone --depth 1 https://github.com/guidovranken/cryptofuzz.git
```

Comment:
Is this still necessary?

Reply:
Eventually we want to clone the tip of Cryptofuzz, right now we're incompatible with the latest version until #129 is merged in. In the meantime for testing I added the checkout a specific commit below that is compatible. I'll add a note to remove it, or just do it if we get #129 merged in today.